style.css

```diff
@@ -538,3 +538,352 @@ input[type=email] {
     margin: 0 1.2rem;
   }
 }
+
+@media screen and (min-width: 768px) {
+  body {
+    max-width: 100vw;
+    min-height: 100vh;
+    overflow-x: hidden;
+    font-size: 20px;
+  }
+
+  main {
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2rem 4rem;
+  }
+
+  .logo {
+    font-size: 1.25rem;
+    line-height: 1;
+    font-weight: 700;
+  }
+
+  .icon {
+    margin: 0 2rem;
+  }
+
+  .navlink {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .navlink a {
+    margin-right: 2rem;
+    font-size: 0.9rem;
+    font-weight: 600;
+    line-height: 1.3;
+  }
+
+  .navlink a i {
+    font-size: 1rem;
+  }
+
+  .hidden {
+    display: block;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .base {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+  }
```

Comment:
- [ ] Good job with the mobile version 👍 Please make sure that the text is aligned according to the Figma template and the text is not overlapping the image. See the images below for more clarity 💡 _Hint: Reduce padding from left and right_
- [ ] Also, please make sure to use the same image for mobile version as given in the design guidelines ✨

Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/236083330-285201ec-fad4-4758-b14a-44d9c201e5a9.png) | ![image](https://user-images.githubusercontent.com/106119105/236083379-d113b2c1-549f-4903-8d9b-732858dc9ba3.png)